packages/sources/alpine/src/endpoint/lastblock.ts

```diff
@@ -0,0 +1,33 @@
+import { Validator, util } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { ethers } from 'ethers'
+import stagingAbi from '../abi/stagingContract.json'
+
+export const supportedEndpoints = ['lastblock']
+
+export const inputParameters: InputParameters = {
+  stagingAddress: true,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const { stagingAddress } = validator.validated.data
+  const network = validator.validated.data.network || config.network
+
+  const rpcUrl =
+    network == 'ETHEREUM' ? util.getEnv('ETHEREUM_RPC_URL') : util.getEnv('POLYGON_RPC_URL')
```

Comment:
Suggestion: Create a custom Config object that extends `{ Config } from '@chainlink/types'` where you can define both Ethereum and Polygon RPC URLs.

This makes it easier to manage the configurable params.

Example (but only for a single network): https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/sources/curve/src/config.ts